src/tests/readLocalStorage.test.js

```diff
@@ -0,0 +1,27 @@
+import readLocalStorage from "../js/api/readLocalStorage";
+
+describe("Testing functions readLocalStorage and writeLocalStorage which executes io-operations with localStorage", () => {
+  const [KEY, VALUE] = ["foo", "bar"];
+
+  beforeEach(() => {
```

Comment:
[здесь](https://www.npmjs.com/package/jest-localstorage-mock) в примере написано Reset your localStorage data and mocks before each test to prevent leaking, но если это важно, то исправил на after